dokka-runners/dokka-gradle-plugin/src/test/kotlin/formats/DokkaFormatTasksTest.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2014-2025 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package org.jetbrains.dokka.gradle.formats
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.collections.shouldBeEmpty
+import io.kotest.matchers.collections.shouldHaveSingleElement
+import io.kotest.matchers.shouldBe
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.apply
+import org.gradle.kotlin.dsl.getByType
+import org.gradle.testfixtures.ProjectBuilder
+import org.jetbrains.dokka.gradle.DokkaBasePlugin
+import org.jetbrains.dokka.gradle.DokkaExtension
+import org.jetbrains.dokka.gradle.dependencies.FormatDependenciesManager
+import org.jetbrains.dokka.gradle.utils.enableV2Plugin
+
+class DokkaFormatTasksTest : FunSpec({
+
+    context("verify format tasks") {
+        val project = projectWithDgpV2()
+
+        val dokkaExtension = project.extensions.getByType<DokkaExtension>()
+
+        val formatName = "testFormatName"
```

Comment:
nit: could this `formatName` variable used in assertions related to it? What do you think if it makes sense?